In [3]:
#Consider the string given below and answer the following questions 
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"
print(len(string))

193


####  Calculate the number of characters in the string

-  193
-  200
-  300
-  500


In [5]:
# Type your code here
print(string.count('a'))

2


#### Count the number of occurences of 'a' in the string

-  1
-  2
-  10
-  5

In [7]:
# Type your code here
string.startswith('if')

False

#### Identify whether the string starts with 'if'.

-  True
-  False

In [8]:
# Type your code here
string[63:88]

'mCfgDYqbaBHAUvIWhMnvwZnEM'

#### Return the substring starting at index 63 and ending at index 88  [Both the indexes are inclusive]

-  'OQYWFClFhFGA'
-  'mCfgDYqbaBHAUvIWhMnvwZnEMV'
-  'IWYwGKpmZhnJiyzTslSIhSwv'
-  'JMEphzmifTkyqOMNpnOtXZ'

In [10]:
# Type your code here
string[45]

'z'

#### Identify the character at index 45

-  a
-  b
-  z
-  x

In [ ]:
# Type your code here